src/app/common/components/CertificateUpload.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { render } from 'react-dom';
+import {
+  Button,
+  InputGroup,
+  TextInput
+} from '@patternfly/react-core';
+
+interface ICertificateUploadProps {
+  isDisabled: boolean;
+  onBlur: any;
+  onChange: any;
+  onInput: any;
+  name: string;
+}
+
+interface ICertificateUploadState {
+  display: string;
+}
+
+class CertificateUpload extends React.Component<ICertificateUploadProps, ICertificateUploadState> {
+
+  private fileInput = React.createRef<HTMLInputElement>();
+
+  constructor(props) {
```

Comment:
Remove constructor when converting to a functional component. We can also use the useState hook for the display value:

`  const [display, setDisplay] = useState('');`